client/apps/edit/components/content/sections/social_embed/test/index.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import configureStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import { mount } from 'enzyme'
+import { NewsArticle } from '@artsy/reaction/dist/Components/Publishing/Fixtures/Articles'
+import { SocialEmbed } from '@artsy/reaction/dist/Components/Publishing/Sections/SocialEmbed'
+import { SectionSocialEmbed } from '../index'
+import { SocialEmbedControls } from '../controls'
+
+describe('Section Social Embed', () => {
+  let props
+
+  beforeEach(() => {
+    props = {
+      article: NewsArticle,
+      section: NewsArticle.sections[2]
+    }
+  })
+
+  const getWrapper = (props) => {
+    const mockStore = configureStore([])
+    const store = mockStore({
+      app: {
+        channel: {}
+      },
+      edit: {
+        article: NewsArticle,
```

Comment:
I usually use 'props.article' here, which gives the ability to change the article data in each test.